airbyte-integrations/connectors/source-intercom/metadata.yaml

```diff
@@ -43,6 +43,7 @@ data:
     - cdk:low-code
     - language:manifest-only
   connectorTestSuitesOptions:
+    - suite: unitTests
```

Comment:
Interesting... unit tests were not executed for this connector

Reply:
@maxi297 - Is this still an issue or did you find a solution?

Reply:
I mean, adding this line allows for the tests to run so we're good. I was just concerned that we weren't running unit tests for source-intercom

Reply:
Got it. Thanks for clarifying. This issue has gotten me before as well - specifically the omitted declaration.